docs/versions.rst

```diff
@@ -14,6 +14,8 @@ TEAL Version PyTeal Version
 2            >= 0.6.0
 3            >= 0.7.0
 4            >= 0.8.0
+5            >= 0.9.1
```

Comment:
v5 support started in v0.9.0 actually
```suggestion
5            >= 0.9.0
```

Reply:
Yes. I thought it was not fully supported. Sorry.